### PR TITLE
Cold Tier

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/src/autorest.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/df70965d3a207eb2a628c96aa6ed935edc6b7911/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-08-06/blob.json
+    - C:\Users\seanmcc\git\azure-rest-api-specs\specification\storage\data-plane\Microsoft.BlobStorage\preview\2021-12-02\blob.json
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/src/autorest.md:7]: TODO fix this when https://github.com/Azure/azure-rest-api-specs/pull/20281 merges

File: sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs [hunk 2]
```diff
@@ -36,7 +36,7 @@ internal static class StorageVersionExtensions
         /// </summary>
         public const ServiceVersion LatestVersion =
 #if BlobSDK || QueueSDK || FileSDK || DataLakeSDK || ChangeFeedSDK
-            ServiceVersion.V2021_10_04;
+            ServiceVersion.V2021_12_02;
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs:39]: I bumped `LatestVersion` for the Cold Tier private preview.  Undo this before this PR merges.

**Review by amnguye — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs/src/autorest.md (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/src/autorest.md]: Fixed.

File: sdk/storage/Azure.Storage.Blobs/src/autorest.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/df70965d3a207eb2a628c96aa6ed935edc6b7911/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-08-06/blob.json
+    - https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/d85f590261f1a8f9d9a1fd07771bc839dc8b6b9e/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json
```
[the remark this answers is not in the document]
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/src/autorest.md]: Fixed.

File: sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs]: Fixed.

**Pull request merged**